modules/node/src/transfer/transfer.repository.ts

```diff
@@ -16,7 +16,7 @@ export class PeerToPeerTransferRepository extends Repository<PeerToPeerTransfer>
 @EntityRepository(LinkedTransfer)
 export class LinkedTransferRepository extends Repository<LinkedTransfer> {
   async findByPaymentId(paymentId: string): Promise<LinkedTransfer | undefined> {
-    return await this.findOne({ where: { paymentId } });
+    return await this.findOne({ relations: ["senderChannel"], where: { paymentId } });
```

Comment:
what does this query change do?

Reply:
Adds the `senderChannel` as an object so you can query `transfer.senderChannel.publicIdentifier` for example.